src/components/molecules/timeline/timeline.tsx

```diff
@@ -20,7 +20,7 @@ export const Timeline = ({ events }: TimelineProps): JSX.Element => (
   <div className="review-timeline">
     <dl className="review-timeline__list">
       {
-        events.map((entry, index) => (
+        events.sort((a, b) => new Date(b.date).getTime() - new Date(a.date).getTime()).map((entry, index) => (
```

Comment:
Switched to this which has the same expected effect but this approach is slightly more robust.